### PR TITLE
Fix devtool strip

### PR DESCRIPTION
**Pull request by gc-plp**
# Reason for This PR

`./devtool strip` was failing due to incorrect path.

## Description of Changes

Fix strip command and add a command called `ci` to use to test the release process end to end.

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

File: tools/devtool [hunk 1]
```diff
@@ -1118,6 +1121,19 @@ cmd_tag() {
     say "Tag v$version created."
 }
 
+# Runs a continuous integration test by running these devtool subcommands:
+# 1. devtool -y test
+# 2. devtool -y build --release
+# 3. devtool strip
+cmd_ci() {
```
Comment (alindima) [inline tools/devtool:1129]: This is to use in our CI pipelines?

File: tools/devtool [hunk 2]
```diff
@@ -775,13 +775,13 @@ cmd_strip() {
       strip $strip_flags\
         "$CTR_CARGO_TARGET_DIR/$target/$profile/firecracker" \
         "$CTR_CARGO_TARGET_DIR/$target/$profile/jailer" \
-        "$CTR_CARGO_TARGET_DIR/$target/$profile/seccompiler/seccompiler"
+        "$CTR_CARGO_SECCOMPILER_TARGET_DIR/$target/$profile/seccompiler"
```
Comment (alindima) [inline tools/devtool:782]: nice catch!

**Review by alindima — APPROVED** [review]

File: tools/devtool (hunk 1 above)
Comment (gc-plp) [reply tools/devtool:1129]: Yep, we should test that a PR also doesn't break the release steps, whatever those steps are.

**Comment by gc-plp** [comment]
Given this a second thought and we need to enable proper flexibility when testing with the `ci` command. I'll circle back on this once I have a better proposal.

I'll remove the `ci` command commit and mark it as ready again.

**Review by acatangiu — APPROVED** [review]

**Review by alindima — APPROVED** [review]

**Pull request merged**